src/App.css

```diff
@@ -57,3 +57,5 @@ html, body {
 .block-item-description:hover{
     cursor: pointer;
 }
+
+/* animated background */
```

Comment:
I found a really neat animated css background for the landing page but couldn't get it to work with react.
https://codepen.io/mohaiman/pen/MQqMyo